src/Request/uapi-parser.js

```diff
@@ -349,4 +351,4 @@ Parser.prototype.parse = function (xml) {
     });
 };
 
-export default Parser;
+module.exports = { Parser, defaultConfig, errorsConfig };
```

Comment:
It's better to keep `defaultConfig` and `errorsConfig` in separate files and reimport it here. So this module will export just `Parser` and other files will be able to import `defaultConfig` and `errorsConfig` as well.